index.bs

```diff
@@ -12269,7 +12269,10 @@ to Consider</a>
 
 	In practice however, this merely allows deduction of information
 	already readily available by easier means (User Agent string),
-	such as "this is Chrome running on an x86".
+	such as "this is Chrome running on an x86".  However, to reduce the
```

Comment:
I'd prefer if we do not name products in an open specification.

> "this is browser X running on platform Y"

Reply:
I didn't actually change that, but I can change it to browser x on y.